oslabs-beta/Kubiquity
# Update IPC

WHAT DOES THIS PR DO?
1. Allows for repeated, automated data queries. 
2. Removes log test files. 

HOW TO TEST THIS PR:
1. Open Docker Desktop. 
2. Run `minikube start`. 
3. Run `gh pr checkout 41 && npm install && npm run dev`.
4. Verify that the app is refreshing K8s data.
